### PR TITLE
fix: events not deleted on cancelling maintenance schedule

### PR DESCRIPTION
**Pull request by scmmishra**
When cancelling a maintenance schedule, associated events would not be deleted. This was because they were not created properly in the first place. 

This PR fixes that

Depends On: https://github.com/frappe/frappe/pull/11221 [Merged]

### CODE REVIEW

**Comment by scmmishra** [comment]
@Mergifyio backport version-12-hotfix

**Comment by scmmishra** [comment]
@marination @nabinhait can we get this merged

**Comment by scmmishra** [comment]
@marination @nabinhait @deepeshgarg007 Need your review on this

File: erpnext/maintenance/doctype/maintenance_schedule/test_maintenance_schedule.py [hunk 1]
```diff
@@ -2,11 +2,47 @@
...
+from frappe.utils.data import get_datetime, add_days
 
 import frappe
 import unittest
 
 # test_records = frappe.get_test_records('Maintenance Schedule')
 
 class TestMaintenanceSchedule(unittest.TestCase):
-	pass
+	def events_should_be_created_and_deleted(self):
```
Comment (marination) [inline erpnext/maintenance/doctype/maintenance_schedule/test_maintenance_schedule.py]:
```suggestion
	def test_events_should_be_created_and_deleted(self):
```

**Review by marination — APPROVED** [review]

**Comment by scmmishra** [comment]
Backport: https://github.com/frappe/erpnext/pull/23219

**Pull request merged**